packages/storybook/src/clinical/organisms/plan-definition-editor/plan-definition-editor.test.tsx

```diff
@@ -0,0 +1,122 @@
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import SelectedPlanDetail, { ISelectedPlanDetailProps } from '@ltht-react/plan-definition-editor'
+import { planDefinition } from './plan-definition-editor.fixtures'
+
+describe('selected plan detail', () => {
+  const defaultProps: ISelectedPlanDetailProps = {
+    onProblemChange: jest.fn(),
+    planDefinition,
+    selectedProblemIds: [],
+  }
+
+  afterEach(() => {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    ;(defaultProps.onProblemChange as jest.MockedFunction<any>).mockClear()

```

Comment:
Uninvited semi-colon guest here can be evicted :P 

Also I could be wrong but I think you could get rid of the cast by creating the mock function on another line, ie. 
`const mockOnProblemChange = jest.fn()`

... `onProblemChange: mockOnProblemChange`

... `mockOnProblemChange.mockClear()`

Or you could just use `jest.clearAllMocks()` since it's the only one in the file

Reply:
Every time I deleted that semi colon and formatted, it returned.. Presumably something in our linting setup. Will go with your implementation, much nicer.